app/models/user.rb

```diff
@@ -2,6 +2,12 @@ class User < ApplicationRecord
   validates :email, uniqueness: true, presence: true
   validates :password, presence: true
 
+  has_many :followed_users, foreign_key: :follower_id, class_name: 'Friend'
```

Comment:
Rubocop wanted us to add :denpendent and :inversed_of but we didn't understand the purpose of it. We will ask the teachers and refactor accordingly